batchflow/utils.py

```diff
@@ -148,3 +149,57 @@ def update_bar(bar, bar_desc, **kwargs):
             desc = str(desc)
         bar.set_description(desc)
     bar.update(1)
+
+def plot_images_predictions(images, targets, proba, ncols=5, classes=None, models_names=None, **kwargs):
```

Comment:
```suggestion
def plot_images(images, labels=None, proba=None, ncols=5, classes=None, models_names=None, **kwargs):
```
When labels and proba are not passed, just show images.
When proba is not passed, show true and predicted labels.